doc/specs/wiki.md

```diff
@@ -0,0 +1,222 @@
+# Wiki Specs
+
+## Metadata
+
+Author: guitarman0831
+
+### Changelog
+
+**2015-07-18**
+
+* Initial version
+
+## Table of contents
+
+1. Motivation
+2. Business rules
+3. Use cases
+4. Data model
+
+## Motivation
+
+>The site contains a variety of users with a vast wealth of knowledge. The wiki is a place where users can put their knowledge to work, and write articles for users to use. These may be supplementary to other documents on the site, or standalone documents such as guides.
+
+## Business rules
+
+> * Articles must be able to be read/write locked based on user class.
+> * Staff members must have full access to articles.
+> * Article creation must be associated with a permission.
+
+## Use cases
+
+These use cases describe various ways the authentication system will interact.
+
+**Use cases:**
+
+### 1.0 Creating an article
+
+**Primary Actor:** A user
+
+**Brief:** Users must be able to create wiki articles in order to share knowledge with other users.
+
+**Preconditions:** The user has the proper permission to create a wiki article.
```

Comment:
must not have the wiki creation permissions disabled or they have the wiki management permissions